metadata-transformer/src/main/java/no/unit/nva/metadata/filters/FilterShorterTitles.java

```diff
@@ -0,0 +1,23 @@
+package no.unit.nva.metadata.filters;
+
+import no.unit.nva.metadata.extractors.ExtractorUtil;
+import no.unit.nva.metadata.type.DcTerms;
+import org.eclipse.rdf4j.model.Literal;
+import org.eclipse.rdf4j.model.Model;
+import org.eclipse.rdf4j.model.Statement;
+import org.eclipse.rdf4j.model.Value;
+
+public final class FilterShorterTitles {
```

Comment:
Add Javadoc for this class, explaining what it does, i.e. filters out titles that are shorter than the supplied statement.